conf/modules.config

```diff
@@ -18,6 +18,10 @@ process {
         saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
     ]
 
+    withName: SEQTK_SAMPLE {
+        ext.args = '-s100'
+    }
```

Comment:
I have no opinion on whether the pipeline should publish the subsampled files. But if you want to output them in the way you describe in the `output.md`, you will probably need to define a corresponding `publishDir` directive here? ([There is also a new possibility called Workflow Output Schema, that I am however not yet familiar with](https://github.com/nf-core/fetchngs/pull/312). Did you use that without me noticing it?)

Reply:
I think is always output because there is a general statement here: https://github.com/nf-core/seqinspector/blob/a2d589f3a2c66e8377a08f572288d0e76ab22daf/conf/modules.config#L15, you mean something different? I think is nice to have the reads that have been actually used in the rest of the analysis